### ISSUE TITLE
Color of h2 element changing to red in output window but giving error!

### ISSUE THREAD

**Issue by amit538** [FreeCodeCamp/FreeCodeCamp#10617]
Challenge [use-a-css-class-to-style-an-element](https://www.freecodecamp.com/en/challenges/html5-and-css/use-a-css-class-to-style-an-element) has an issue.
User Agent is: <code>Mozilla/5.0 (X11; Linux x86_64; rv:48.0) Gecko/20100101 Firefox/48.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```html
    
    
<style>
.red-text {
  color: red;
}
</style>

<h2 class="red-text">CatPhotoApp</h2>

<p>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>

    ```

**Comment by niconiahi** [comment]
same prob here, fix please

**Comment by dhcodes** [comment]
Closed as duplicate of #10595

**Comment by dhcodes** [comment]
Thanks for the issue!
